Refactoring person to customer

ZhangCX10032 opened the pull request:
Rename images 
   - Rename DeletePersonSdForLogic.png to DeleteCustomerSdForLogic.png
    - Rename SDforDeletePerson.png to SDforDeleteCustomer.png

Update DeveloperGuide
    - update the instructions of person to customer

Update UserGuide
    - update the instruction of add Customer, delete Customer
      and edit Customer

Update Codes
    - Change the name of all functions&classes from person to customer

Update Tests
    - Change the name of tests from person to customer

Commit:
Rename images
    - Rename DeletePersonSdForLogic.png to DeleteCustomerSdForLogic.png
    - Rename SDforDeletePerson.png to SDforDeleteCustomer.png

Update DeveloperGuide
    - update the instructions of person to customer

Update UserGuide
    - update the instruction of add Customer, delete Customer
      and edit Customer

Update Codes
    - Change the name of all functions&classes from person to customer

Update Tests
    - Change the name of tests from person to customer

Tejas2805 reviewed (dismissed):
Refactoring looks good but there a few code style errors like import statements having *. They have a preference for having imports explicitly. The method names have an error. Please do make these changes on the basis of code style requirements. Otherwise all looks good

Commit:
Improve the style according to the feedback of Codacy
    - rename methods by deleting the underscores in
        - UniqueCustomerListTest.java
        - UJsonAdaptedCustomerTest.java
        - CustomerListPanelTest.java
    - refactor the methods in UJsonAdaptedCustomer.java
    - change "equalTO(null)" to "== null" in CustomerCardTest.java

ZhangCX10032 committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	docs/UserGuide.adoc
#	src/test/java/seedu/address/model/customer/CustomerTest.java
#	src/test/java/seedu/address/testutil/TypicalCustomers.java

ZhangCX10032 committed:
Revert "Merge remote-tracking branch 'upstream/master'"

This reverts commit 636e30b162a05696381b676e3e995629d147cad4.

ZhangCX10032 committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-t12-1/main

# Conflicts:
#	docs/UserGuide.adoc
#	src/test/java/seedu/address/model/customer/CustomerTest.java
#	src/test/java/seedu/address/testutil/TypicalCustomers.java

ZhangCX10032 committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-t12-1/main into ZhangCX10032-master

# Conflicts:
#	docs/UserGuide.adoc
#	src/test/java/seedu/address/model/customer/CustomerTest.java
#	src/test/java/seedu/address/testutil/TypicalCustomers.java

ZhangCX10032 committed:
Fixed the bugs when caused during the merging

ZhangCX10032 committed:
Rename the VIP_PERSON to VIP_CUSTOMER

ZhangCX10032 committed:
Merge branch 'ZhangCX10032-master'

# Conflicts:
#	docs/UserGuide.adoc
#	src/test/java/seedu/address/model/customer/CustomerTest.java
#	src/test/java/seedu/address/testutil/TypicalCustomers.java

ZhangCX10032 committed:
Improve the style according to the feedback of Travis CI

ZhangCX10032 committed:
Improve the style according to the feedback of Travis CI

ZhangCX10032 committed:
Improve the style according to the feedback of Travis CI

ZhangCX10032 committed:
no message

ZhangCX10032 committed:
rename testNames as required by codacy

ZhangCX10032 commented:
> Refactoring looks good but there a few code style errors like import statements having *. They have a preference for having imports explicitly. The method names have an error. Please do make these changes on the basis of code style requirements. Otherwise all looks good

Thanks for reviewing. I have corrected those style errors.
